employee.rb

```diff
@@ -0,0 +1,30 @@
+class Employee
+  attr_accessor :name, :salary, :reviews, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+    @satisfactory = []
+  end
+
+  def employee_review(review)
+    @reviews << review
+  end
+
+  def satisfactory_rating
+    @satisfactory = false
+  end
+
+  def unsatisfactory_rating
+    @unsatisfactory = true
```

Comment:
I think that you got a little confused about your instance variables here.  You have BOTH a @satisfactory and an @unsatisfactory, which could lead to bad states.  For instance, what does it mean if they're both true?

In any case, the best approach is to probably have the satisfactory_rating method set @satisfactory = true, and the unsatisfactory_rating method set @satisfactory = false.